MetaMask/metamask-extension
# Rename reset account settings

## Explanation
Rename "reset account" to something more accurate.

This PR will change "reset account" in settings to "Clear activity and nonce data"

* Fixes #14418 
## Screenshots/Screencaps



### Before

https://user-images.githubusercontent.com/97883527/215073676-e1aab120-dc5a-434f-affe-3f792c63b54a.mov




### After

https://user-images.githubusercontent.com/97883527/215073113-9e1119f6-57df-4f33-884a-2aaca3f51478.mov





## Manual Testing Steps

Go to Settings 
Go to Advance
Check "Reset account" option is renamed to "Clear activity and nonce data"



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
